### PR TITLE
myBinder (0.28)

### PR DESCRIPTION
**Pull request by alixdamman**
The last commit will be dropped before to squash and merge. It is just for testing the mybinder badge.

http://larray-test.readthedocs.io/en/documentation/index.html

### CODE REVIEW

File: doc/source/conf.py [hunk 1]
```diff
@@ -83,11 +83,9 @@
         
     .. nbinfo::
         Interactive online version:
-        :raw-html:`<a href="https://mybinder.org/v2/gh/liam2/larray/{{ env.config.release }}?
+        :raw-html:`<a href="https://mybinder.org/v2/gh/alixdamman/larray/{{ env.config.release }}?
```
Comment (gdementen) [inline doc/source/conf.py]: ?

File: doc/source/conf.py [hunk 2]
```diff
@@ -115,7 +113,7 @@
 # The short X.Y version.
 version = larray.__version__
 # The full version, including alpha/beta/rc tags.
-release = larray.__version__
+release = 'documentation' #larray.__version__
```
Comment (gdementen) [inline doc/source/conf.py]: Out of curiosity, why do you need this change?

File: doc/source/conf.py (hunk 1 above)
Comment (alixdamman) [reply doc/source/conf.py]: read the commit message

File: doc/source/conf.py (hunk 2 above)
Comment (alixdamman) [reply doc/source/conf.py]: read the commit message

File: doc/source/conf.py (hunk 1 above)
Comment (gdementen) [reply doc/source/conf.py]: Ok, clear now. 

PS: I usually do not read individual commits, unless you somehow hint at it in the PR notes... which I just noticed you actually **did**. I just somehow missed that comment. Sorry about this.

**Pull request merged**